build.gradle

```diff
@@ -252,6 +252,33 @@ tasks.withType(JavaCompile) {
 	options.encoding = 'UTF-8'
 }
 
+task pruneBin(type: Delete) {
+	onlyIf {
+		file('build/releases/KoLmafia').exists()
+	}
+	delete 'build/releases/KoLmafia'
+}
+
+task bin(type: Exec) {
+	dependsOn shadowJar
+	outputs.files file('build/releases/')
+
+	executable 'jpackage'
+	doFirst {
```

Comment:
Would recommend a gradle plugin instead of using exec (even if it is just calling the exec).
e.g. : https://github.com/petr-panteleyev/jpackage-gradle-plugin or https://github.com/xcporter/jpkg

Reply:
Done.